openedx_webhooks/config.py

```diff
@@ -10,16 +10,18 @@ class DefaultConfig:
     GITHUB_WEBHOOKS_SECRET = os.environ.get("GITHUB_WEBHOOKS_SECRET")
     JIRA_OAUTH_CONSUMER_KEY = os.environ.get("JIRA_OAUTH_CONSUMER_KEY")
     JIRA_OAUTH_RSA_KEY = os.environ.get("JIRA_OAUTH_RSA_KEY")
-    CELERY_ACCEPT_CONTENT = ["json"]
-    CELERY_TASK_SERIALIZER = "json"
-    CELERY_RESULT_SERIALIZER = "json"
-    CELERY_EAGER_PROPAGATES_EXCEPTIONS = True
-    CELERY_BROKER_URL = os.environ.get("REDIS_URL", "redis://")
-    CELERY_RESULT_BACKEND = os.environ.get("REDIS_URL", "redis://")
+
+    # Celery Config Settings
+    accept_content = ["json"]
+    task_serializer = "json"
+    result_serializer = "json"
+    task_eager_propagates = True
+    broker_url = os.environ.get("REDIS_URL", "redis://")
+    celery_result_backend = os.environ.get("REDIS_URL", "redis://")
```

Comment:
yep, it felt just as wrong to me as well but sadly it's true: https://docs.celeryq.dev/en/stable/userguide/configuration.html#new-lowercase-settings

Technically we don't need to do most of this till next version when the old ones will break but since I was already in here, I just did them with this upgrade.